server/src/main/java/demo/HelloServerApplication.java

```diff
@@ -0,0 +1,39 @@
+package demo;
+
+import java.util.List;
+import java.util.Random;
+
+import reactor.core.publisher.Mono;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.client.ServiceInstance;
+import org.springframework.cloud.client.discovery.DiscoveryClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author Olga Maciaszek-Sharma
+ */
+@SpringBootApplication
+@RestController
+public class HelloServerApplication {
+
+	@Autowired
+	DiscoveryClient client;
```

Comment:
How is this working without eureka or another?

Reply:
It assumes there's Eureka - it's there in the README, along with a Eureka server sample project link.

Reply:
Ah, assumes eureka running. https://github.com/spring-cloud-samples/spring-cloud-square-retrofit-webclient/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R4